pubsub/pulsar/pulsar.go

```diff
@@ -151,7 +151,7 @@ func (p *Pulsar) handleMessage(msg pulsar.ConsumerMessage, handler func(msg *pub
 
 	return pubsub.RetryNotifyRecover(func() error {
 		p.logger.Debugf("Processing Pulsar message %s/%#v", msg.Topic(), msg.ID())
-		err := handler(&pubsubMsg)
+		err := handler(context.Background(), &pubsubMsg)
```

Comment:
Here you can use `p.ctx` instead of `context.Background()`.

Reply:
Done 👍